full-detector-simulations/FCCeeCLD/FCCeeCLD.md

```diff
@@ -23,3 +23,678 @@ source /cvmfs/sw-nightlies.hsf.org/key4hep/setup.sh
+
+<details>
+
+<summary><b>Click to show the code of the python script <code>dd4hep2root.py</code>. </b></summary>
+
+```python
+#!/usr/bin/env python3
+
+import sys
+import argparse
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Convert detector')
+    parser.add_argument('-c', '--compact', help='Compact file location(s)',
+                        required=True, type=str, nargs='+')
+    parser.add_argument('-o', '--out', help='Converted file path',
+                        default='detector.root', type=str)
+    args = vars(parser.parse_args())
+
+    convert(args['compact'], args['out'])
+
+
+def convert(compact_files, out_path):
+    print('INFO: Converting following compact file(s):')
+    for cfile in compact_files:
+        print('      ' + cfile)
+
+    import ROOT
+
+    ROOT.gSystem.Load('libDDCore')
+    description = ROOT.dd4hep.Detector.getInstance()
+    for cfile in compact_files:
+        description.fromXML(cfile)
+
+    ROOT.gGeoManager.SetVisLevel(9)
+    ROOT.gGeoManager.SetVisOption(0)
+    ROOT.gGeoManager.Export(out_path)
+
+
+if __name__ == '__main__':
+    main()
+
+```
+
+</details>
+
+<br/>
+
+2. Using Qt interface of Geant4, accesible via `runType` argument of `ddsim`
+
+<details>
+
+<summary><b> Click to show how to display the detector geometry using Geant4. </b></summary>
+
+```shell
+source /cvmfs/sft.cern.ch/lcg/views/dev4/latest/x86_64-centos7-gcc11-opt/setup.sh
+ddsim --compactFile description.xml --runType qt
+```
+
+After building internally the geometry, a window will open. The menu on the left gives access to the builtin Geant4 macros. In order to show the geometry, we have to type some commands:
+
+```
+# Use this open statement to create an OpenGL view:
+/vis/open OGL 600x600-0+0
+
+# Draw geometry:
+/vis/drawVolume
+
+# Specify view angle:
+/vis/viewer/set/lightsVector -1 0 0
+/vis/viewer/set/viewpointVector -1 0 0
+```
+
+
+</details>
+:::
+
+<br/>
+
+
+### Simulate Muons in CLD detector
+
+To simulate the interaction of 100 muons with the CLD detector, the following command is used inside the directory `mydd4heptutorial`. It takes some minutes to build the geometry and run the actual simulation. 
+
+```shell
+ddsim --compactFile FCCee_o1_v05/FCCee_o1_v05.xml \
+      --enableGun \
+      --gun.distribution uniform \
+      --gun.energy "10*GeV" \
+      --gun.particle mu- \
+      --numberOfEvents 100 \
+      --outputFile Step1_edm4hep.root
+```
+
+And now plot the distribution with `showPlots.py`
+
+```shell
+python showPlots.py Step1_edm4hep.root
+```
+
+### Removing an included XML file
+
+The detector geometry is complex, and takes a while to be built internally. To speed up the following steps, the Silicon Tracker can be removed. To do so, 
```

Comment:
```suggestion
The detector geometry is complex, and takes a while to be built. To speed up the following steps, the Silicon Tracker can be removed. To do so, 
```

It kind of makes sense to give the warning at the start and drop the tracker before any first simulation